adminpages/orders.php

```diff
@@ -1441,6 +1512,15 @@ class="alternate"<?php } ?>>
 								),
 							];
 
+							if( pmpro_allowed_refunds( $order ) ) {
+								$actions['refund'] = sprintf(
+									'<a title="%1$s" href="%2$s">%3$s</a>',
+									esc_attr__( 'Refund', 'paid-memberships-pro' ),
+									'javascript:pmpro_askfirst(\'' . esc_js( $refund_text ) . '\', \'' . esc_js( $refund_nonce_url ) . '\'); void(0);',
```

Comment:
Please confirm this change works as expected, there's a possibility that the wp_json_encode() usage having double quotes could interfere with how this is meant to work -- but it should theoretically work as we need it to.

```suggestion
									esc_js( 'javascript:pmpro_askfirst(' . wp_json_encode( $refund_text ) . ', ' . wp_json_encode( $refund_nonce_url ) . '); void(0);' ),
```